.changeset/brown-ducks-beg.md

```diff
@@ -0,0 +1,5 @@
+---
+'@chainlink/superstate-adapter': patch
```

Comment:
minor

Reply:
you think? It'll still work

Reply:
minor means you can't go back once upgrade, patch means you can

Reply:
(not important)

Reply:
i don't feel strongly either way 🤷. I guess it does affect the functionality so i'll use minor